preprocessing/lib/csvToJson.js

```diff
@@ -0,0 +1,27 @@
+const csvLoader = require('../lib/load-data');
```

Comment:
we either need a README in lib, or (because the files are not necessarily related) every file might start with a preamble that documents what the file is for